identifiers/country-mx/districts.csv

```diff
@@ -0,0 +1,300 @@
+ocd-division/country:mx/state:ag/fed:1,I Federal Electoral District of Aguascalientes
+ocd-division/country:mx/state:ag/fed:2,II Federal Electoral District of Aguascalientes
+ocd-division/country:mx/state:ag/fed:3,III Federal Electoral District of Aguascalientes
+ocd-division/country:mx/state:bc/fed:1,I Federal Electoral District of Baja California
+ocd-division/country:mx/state:bc/fed:2,II Federal Electoral District of Baja California
+ocd-division/country:mx/state:bc/fed:3,III Federal Electoral District of Baja California
+ocd-division/country:mx/state:bc/fed:4,IV Federal Electoral District of Baja California
+ocd-division/country:mx/state:bc/fed:5,V Federal Electoral District of Baja California
+ocd-division/country:mx/state:bc/fed:6,VI Federal Electoral District of Baja California
+ocd-division/country:mx/state:bc/fed:7,VII Federal Electoral District of Baja California
+ocd-division/country:mx/state:bc/fed:8,VIII Federal Electoral District of Baja California
+ocd-division/country:mx/state:bs/fed:1,I Federal Electoral District of Baja California Sur
+ocd-division/country:mx/state:bs/fed:2,II Federal Electoral District of Baja California Sur
+ocd-division/country:mx/state:cm/fed:1,I Federal Electoral District of Campeche
+ocd-division/country:mx/state:cm/fed:2,II Federal Electoral District of Campeche
+ocd-division/country:mx/state:cs/fed:1,I Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:2,II Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:3,III Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:4,IV Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:5,V Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:6,VI Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:7,VII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:8,VIII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:9,IX Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:10,X Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:11,XI Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:12,XII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:13,XIII Federal Electoral District of Chiapas
```

Comment:
According to https://en.wikipedia.org/wiki/Federal_electoral_districts_of_Mexico, we only have 12 fed districts in Chiapas.

Reply:
Your list does match with the INE list http://cartografia.ife.org.mx/sige7/?distritacion=federal though, which I assume is the authoritative source for this.